### ISSUE TITLE
[EN][Druid][Spell] Dual spec and MotW

### ISSUE THREAD

**Issue by Iambugged** [Atlantiss/BugTracker#2087]
When switching talents as Druid you lose Mark of the Wild buff.
http://www.wowhead.com/spell=1126/mark-of-the-wild

**Comment by MasterK15** [comment]
Everyone loose all buffs when respec... Or im wrong...

**Comment by Glare** [comment]
You are wrong, at the moment you only lose your own buffs but I don't know if it's blizzlike or not.
